api/host.py

```diff
@@ -90,7 +90,7 @@ def add_host_list(body):
                         logger.info("Tags from an HTTP request were ignored")
 
                     input_host = deserialize_host_http(host)
-                    (output_host, add_result) = _add_host(input_host)
+                    (output_host, _, add_result) = _add_host(input_host)
```

Comment:
Here, we can retrieve the Host ID. In contrast to the Insights ID that is not useful here.

```suggestion
                    output_host, host_id, _, add_result = _add_host(input_host)
```